bucket/wemod.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "8.6.0",
+    "description": "Thousands of free mods and trainers for your favorite single-player PC games — all in one place.",
```

Comment:
```suggestion
    "description": "Thousands of free mods and trainers for your favorite single-player PC games — all in one place",
```

Reply:
I find that you has pull a commit to modify this. Thanks.